truss/templates/control/control/helpers/truss_patch/model_code_patch_applier.py

```diff
@@ -0,0 +1,37 @@
+import logging
+from pathlib import Path
+
+# TODO(pankaj) In desparate need of refactoring into separate library
+try:
```

Comment:
let's not add this please. The current solution is that the import path is added to the tests. see `test_server.py`. I'd rather not introduce something like this if we don't need to until the refactor which we're starting to plan.

Reply:
This is how you're doing it somewhere else to solve the same problem. I think we just stick with this until we refactor:
https://github.com/basetenlabs/truss/blob/main/truss/tests/templates/control/control/test_server.py#L10

Reply:
ah, I see what you meant now. will do.

Reply:
This doesn't work here. Action and ModelCodePatch types are already available from `truss.templates.control.control.helpers.types` when this code is called from truss library. Adding `/truss/templates/control/control` to path creates duplicate definitions of these types and cause confusion before, `action in [Action.ADD, Action.UPDATE]` fails because the two Action definitions are considered different.

Reply:
I see your concern. I tried. Running out of simple ideas. We may be able to fix this by going down to importlib layer, but I'm not sure that's worth it we want to refactor this out into smaller library soon.